src/bin/rugs_metadata_server.rs

```diff
@@ -216,6 +225,44 @@ mod tests {
         Ok(pool)
     }
 
+    /// Test reading auth config secrets from files
+    #[tokio::test]
+    async fn config_secrets() -> Result<()> {
+        std::fs::write("./rugs_user_auth", USER_AUTH).expect("Could not write user secret to file");
+        std::fs::write("./rugs_ci_auth", CI_AUTH).expect("Could not write CI secret to file");
+
+        let cur_user_auth_file = std::env::var(USER_AUTH_FILE_KEY);
```

Comment:
I appreciate trying to be a good citizen and returning the variables to their previous value, but since the environment variable only affects the current test run, none of the tests should be relying on a value being persisted here -- so it's safe to set the environment variable unconditionally and just remove it after the test finishes.